tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/LocaleCompletionCounter.java

```diff
@@ -1,119 +1,49 @@
 package org.unicode.cldr.web.api;
 
 import com.ibm.icu.dev.util.ElapsedTimer;
-import java.util.ArrayList;
-import java.util.List;
+import java.util.EnumSet;
 import java.util.logging.Logger;
-import org.unicode.cldr.test.CheckCLDR;
-import org.unicode.cldr.test.CoverageLevel2;
-import org.unicode.cldr.test.SubmissionLocales;
-import org.unicode.cldr.test.TestCache;
 import org.unicode.cldr.util.*;
-import org.unicode.cldr.web.STFactory;
-import org.unicode.cldr.web.SurveyLog;
-import org.unicode.cldr.web.SurveyMain;
+import org.unicode.cldr.web.*;
 
 public class LocaleCompletionCounter {
 
     private static final Logger logger = SurveyLog.forClass(LocaleCompletionCounter.class);
 
+    private static final EnumSet<VettingViewer.Choice> choiceSet = EnumSet.of(
+        VettingViewer.Choice.error,
+        VettingViewer.Choice.hasDispute,
+        VettingViewer.Choice.notApproved,
+        VettingViewer.Choice.missingCoverage
+    );
+
     private final String localeId;
     private final Level level;
-    private final CheckCLDR.Options options;
-    private final TestCache.TestResultBundle checkCldr;
-    private final CLDRFile file;
-    private final CLDRFile baselineFile;
     private final LocaleCompletion.LocaleCompletionResponse lcr;
-    private final CoverageLevel2 covLeveller;
-    private final List<CheckCLDR.CheckStatus> results;
+    private final VettingViewer<Organization> vv;
+    private final VettingViewer.DashboardArgs args;
```

Comment:
this is a re-use of DashboardArgs for locale completion, not actually for dashboard -- it should be renamed to something more general in a subsequent pr